src/libinjector/injector.c

```diff
@@ -706,9 +716,9 @@ static event_response_t wait_for_injected_process_cb(drakvuf_t drakvuf, drakvuf_
     injector->detected = true;
 
     if ( injector->break_loop_on_detection )
-        drakvuf_interrupt(drakvuf, 1);
+        drakvuf_interrupt(drakvuf, SIGHUP);
```

Comment:
Instead of `SIGHUP` I would say `SIGINT` is more appropriate. I know `SIGHUP` is `1` but if we are to use standard signal numbers now might as well use one that makes more sense :)

Reply:
OK. I have thought about this, but didn't decide to do it.